### PR TITLE
Fix/icon font awesome print queue

### PR DESCRIPTION
**Pull request by astroidex**
- new icon for file pdf
- gear spin reactivate
- trash to delete

### CODE REVIEW

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js [hunk 1]
```diff
@@ -159,12 +159,12 @@ $.widget("mapbender.mbPrintClientJobList", {
                 .attr('target', '_blank')
                 .attr('title', Mapbender.trans('mb.print.printclient.joblist.open'))
             ;
-            $icon = $('<i/>').addClass('fa fa-file-pdf-o');
+            $icon = $('<i/>').addClass('fa far fa-file-pdf fa-lg');
```
Comment (Phocacius) [inline src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:162]: fa is superflous after adding "far", can be deleted

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js [hunk 2]
```diff
@@ -159,12 +159,12 @@ $.widget("mapbender.mbPrintClientJobList", {
...
             $a.append($icon);
             $group.append($a);
             buttonsEmpty = false;
         } else {
-            loader = '<span class="loading"><i class="fas fa-gear"></i></span>';
+            loader = '<span class="loading"><i class="fa fas fa-gear fa-spin fa-lg"></i></span>';
```
Comment (Phocacius) [inline src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:167]: same here

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js [hunk 3]
```diff
@@ -178,7 +178,7 @@ $.widget("mapbender.mbPrintClientJobList", {
                 .attr('data-id', row.id)
                 .attr('title', Mapbender.trans(deleteTitle))
             ;
-            $icon = $('<i/>').addClass('fa fa-remove');
+            $icon = $('<i/>').addClass('fa far fa-trash-can');
```
Comment (Phocacius) [inline src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:181]: and same here

**Review by Phocacius — CHANGES_REQUESTED** [review]

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js (hunk 1 above)
Comment (legsforwalking) [reply src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:162]: done

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js (hunk 2 above)
Comment (legsforwalking) [reply src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:167]: done

File: src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js (hunk 3 above)
Comment (legsforwalking) [reply src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js:181]: done

**Pull request merged**